Classes/Domain/AbstractAggregateRoot.php

```diff
@@ -110,7 +104,7 @@ protected function __construct(string $identifier)
      */
     final protected function apply(EventInterface $event)
     {
-        $method = sprintf('when%s', $this->eventTypeService->getEventShortType($event));
+        $method = sprintf('when%s', (new \ReflectionClass($event))->getShortName());
```

Comment:
not introduced by this PR, but in general it should be `$methodName` and not `$method` (`$method` would mean, for me, that it contains a `ReflectionMethod` instance)